### ISSUE TITLE
flake version doesn't build anymore

### ISSUE THREAD

**Issue by MaxSchlueter** [vlaci/nix-doom-emacs#99]
The flake doesn't build anymore, because it pulls in the latest doom emacs sources and these are out of sync with the nix-integration.patch. A workaround for me was to explicitly set the revision:

``` nix
doom-emacs.url = "github:hlissner/doom-emacs/f7293fb67ef701244a421fd3bfc04b8e12689edc";
doom-emacs.flake = false;
nix-doom-emacs.url = "github:vlaci/nix-doom-emacs";
nix-doom-emacs.inputs.doom-emacs.follows = "doom-emacs";
```

I think the lock file should be committed in order to point the inputs in the flake to specific revisions.

**Comment by vlaci** [comment]
I have thought about it and I am not sure if such an extension module should fix their dependencies in a lock file or is it the consumer side's responsibility.

**Comment by MaxSchlueter** [comment]
I understand your point, I would still opt for locked revisions though, even though I also don't know of any better way than to manually unpin the inputs in the `flake.nix` file, I feel like it's less of a hassle than finding out that the expression is not building anymore. Maybe the `flake.lock` file could then be used as a replacement for `sources.nix`?

**Comment by vlaci** [comment]
The https://github.com/vlaci/nix-doom-emacs/pull/117 PR is working on exactly that.

**Comment by vlaci** [comment]
I have merged #117 and I hope that the weekly automated jobs will keep the lock file in sync.
